### PR TITLE
doc: Add multitech basic station guide

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

- Add lbs guide for multitech AEP
- Improve regular guide, discourage udp packet forwarder

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in `README.md`.
- [x] Style Guidelines: Documentation obeys style guidelines in `README.md`.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/multitechconduit/_index.md [hunk 1]
```diff
@@ -4,81 +4,94 @@ description: ""
...
+Finally, click **Save and Apply** to apply your configuration.
 
-- You will be prompted to confirm the restart. Choose **OK** to proceed.
+At this point, the gateway will reboot. It will come back online as a DHCP client, so you can disconnect it from your computer, and connect the gateway directly to your router or local network.
 
-{{< figure src="012_System_Confirmation_Prompt.png" alt="Configuration restart prompt" >}}
+## Connecting the Gateway to {{% tts %}}
 
-This will apply the custom settings and reboot the gateway. If all the steps have been followed correctly, your gateway will now connect to {{%tts%}}.
+The Multitech Conduit supports {{% lbs %}} and the legacy UDP packet forwarder. {{% lbs %}} is more secure and supports configuration of custom channel plans, amongst other improvements. {{% tts %}} supports {{% lbs %}}, so please follow instructions for [connecting the Multitech Conduit with {{% lbs %}}]({{< relref "lbs" >}}).
```
Comment (KrishnaIyer) [inline doc/content/gateways/multitechconduit/_index.md:92]: Please add a recommendation to use LBS

File: doc/content/gateways/multitechconduit/lbs.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Configure LoRa Basics™ Station"
+description: ""
+---
+
+This section contains instructions for connecting the Multitech Conduit AEP to {{% tts %}} using {{% lbs %}}.
```
Comment (KrishnaIyer) [inline doc/content/gateways/multitechconduit/lbs.md:6]: `mLinux`?

File: doc/content/gateways/multitechconduit/udp.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Configure UDP Packet Forwarder"
+description: ""
+---
+
+This section contains instructions for connecting the Multitech Conduit AEP to {{% tts %}} using the UDP Packet Forwarder.
```
Comment (KrishnaIyer) [inline doc/content/gateways/multitechconduit/udp.md:6]: IDEM

File: doc/content/gateways/multitechconduit/_index.md [hunk 4]
```diff
@@ -4,81 +4,94 @@ description: ""
...
+![MultiTech MultiConnect Conduit AEP Gateway](multitech-conduit-ap.png)
 
 ## Prerequisites
 
 1. User account on {{% tts %}} with rights to create Gateways.
-2. Multitech Conduit AEP Gateway connected to the internet and running the [latest firmware](http://www.multitech.net/developer/downloads/).
-
-> This article addresses the minimum configuration required to connect your Multitech Conduit AEP model to {{% tts %}}.
-> For the first-time hardware and software setup, you can refer to the appropriate configuration guide on the [Multitech documentation](http://www.multitech.net/developer/products/multiconnect-conduit-platform/conduit/) page.
+2. Multitech Conduit AEP Gateway running [mPower 5.30 firmware or later](http://www.multitech.net/developer/downloads/).
```
[the remark this answers is not in the document]
Comment (benolayinka) [reply doc/content/gateways/multitechconduit/_index.md:20]: mLinux has a separate guide https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/24

File: doc/content/gateways/multitechconduit/_index.md (hunk 1 above)
Comment (benolayinka) [reply doc/content/gateways/multitechconduit/_index.md:92]: This is a recommendation to use LBS

**Review by KrishnaIyer — APPROVED** [review]

**Pull request merged**